app/src/main/java/mozilla/lockbox/view/RootActivity.kt

```diff
@@ -51,4 +57,18 @@ class RootActivity : AppCompatActivity() {
             super.onBackPressed()
         }
     }
+
+    fun showToastNotification(@StringRes strId: Int? = null, text: String? = null, viewGroup: ViewGroup) {
+        assertOnUiThread()
+        val toast = Toast(this)
+
+        toast.duration = Toast.LENGTH_SHORT
+        toast.view = layoutInflater.inflate(R.layout.toast_view, viewGroup, false)
+        toast.setGravity(Gravity.FILL_HORIZONTAL or Gravity.BOTTOM, 0, 0)
+
+        val view = toast.view.findViewById(R.id.message) as TextView
+        view.text = text?.plus(" deleted.") ?: resources.getString(strId!!)
```

Comment:
Nit: localization. 

`deleted` needs to be localizable.
`${thing} deleted` needs to be localizable. This is best done by embedding a `%1$@` (I can never remember the incantation) in the `string.xml` localization.

This will crash if neither the `text` nor the `strId` is present.

Reply:
"deleted" hasn't been localized.. I should reach out to Pike about that